### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by demony**
Homework 1 completed

### CODE REVIEW

File: libs/com/epam/tat/module4/calculator/1.0/calculator-1.0.pom [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.epam.tat.module4</groupId>
+  <artifactId>calculator</artifactId>
+  <version>1.0</version>
+  <description>POM was created from install:install-file</description>
+</project>
```
Comment (komyagin) [inline libs/com/epam/tat/module4/calculator/1.0/calculator-1.0.pom]: For what this file is for?

File: libs/com/epam/tat/module4/calculator/maven-metadata-local.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  <groupId>com.epam.tat.module4</groupId>
+  <artifactId>calculator</artifactId>
+  <versioning>
+    <release>1.0</release>
+    <versions>
+      <version>1.0</version>
+    </versions>
+    <lastUpdated>20210805145652</lastUpdated>
+  </versioning>
+</metadata>
```
Comment (komyagin) [inline libs/com/epam/tat/module4/calculator/maven-metadata-local.xml]: For what this file is for?

File: pom.xml [hunk 3]
```diff
@@ -12,6 +12,72 @@
...
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>${testng.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.20.0</version>
```
Comment (komyagin) [inline pom.xml]: If you use the property for the testng why don't use it for the rest?

**Review by komyagin — CHANGES_REQUESTED** [review]

File: libs/com/epam/tat/module4/calculator/1.0/calculator-1.0.pom (hunk 1 above)
Comment (demony) [reply libs/com/epam/tat/module4/calculator/1.0/calculator-1.0.pom]: Fixed.

File: libs/com/epam/tat/module4/calculator/maven-metadata-local.xml (hunk 2 above)
Comment (demony) [reply libs/com/epam/tat/module4/calculator/maven-metadata-local.xml]: Fixed.

File: pom.xml (hunk 3 above)
Comment (demony) [reply pom.xml]: Fixed.

**Review by komyagin — APPROVED** [review]

**Pull request merged**